src/server/status_server/mod.rs

```diff
@@ -677,6 +783,9 @@ where
                             (Method::GET, "/debug/pprof/heap") => {
                                 Self::dump_heap_prof_to_resp(req).await
                             }
+                            (Method::GET, "/debug/pprof/cmdline") => Self::get_cmdline(req).await,
```

Comment:
From the get_cmdline implementation, it seems it should not be put under /debug/pprof as it's simply return the cmdline arguments, not related to pprof.

Reply:
Check https://gperftools.github.io/gperftools/pprof_remote_servers.html, `cmdline` is also a standard API for pprof